.clang-tidy

```diff
@@ -27,6 +27,7 @@ Checks: >
   -readability-function-cognitive-complexity,
   -readability-function-size,
   -readability-identifier-length,
+  -readability-identifier-naming,
```

Comment:
We can't remove this check. It's one of the most important checks we're using.

Reply:
Ah, sorry, I didn't realize its impact.

Reply:
See those readability-identifier-naming checks farther down the .clang-tidy file? They define a lot of important style preferences.